modules/apps/analytics/analytics-settings-web/package.json

```
@@ -0,0 +1,11 @@
+{
+	"main": "js/index.es.js",
+	"name": "@liferay/analytics-settings-web",
+	"private": true,
+	"scripts": {
+		"build": "liferay-npm-scripts build",
+		"checkFormat": "liferay-npm-scripts check",
+		"format": "liferay-npm-scripts fix"
+	},
+	"version": "5.0.34"
```

Since this is a brand new module, the version should start **1.0.0**

```suggestion
	"version": "1.0.0"
```

---

I set this to version 1.0.0, but source formatter is setting it back :thinking: